ui/src/pages/View.js

```diff
@@ -17,9 +17,106 @@
 import React, {Component} from 'react';
 import {FormattedMessage, defineMessages, injectIntl} from 'react-intl';
 
+import Footer from './../components/Footer.js';
 import LoadingIndicator from './../components/LoadingIndicator.js';
+import RepoHeader from './../components/RepoHeader.js';
+import SearchBar from './../components/SearchBar.js';
 import Utils from './../Utils.js';
 
+// TODO(gimite): Consider sharing some messages with Create.js because they
+// have many overlaps.
+const MESSAGES = defineMessages({
+  age: {
+    id: 'View.age',
+    defaultMessage: 'Age',
+    description: 'A label for a person\'s age.',
+  },
+  authorOfThisRecord: {
+    id: 'View.authorOfThisRecord',
+    defaultMessage: 'Author of this record',
+    description: ('The title of a section in a person record which contains '
```

Comment:
Since these descriptions are going to get sent to translators, it might be good to avoid descriptions that refer to "person records" since a translator's unlikely to know quite what that means. Maybe something like "The title of a section with information about the source/author of the rest of the information on the page."